src/tox_lsr/config_files/tox-default.ini

```diff
@@ -314,3 +314,15 @@ deps =
 commands =
     curl -L -o {envdir}/report-modules-plugins.py https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/master/report-modules-plugins.py
     python {envdir}/report-modules-plugins.py --details .
+
+[testenv:ansible-managed-var-comment]
+changedir = {toxinidir}
+basepython = python3
+whitelist_externals =
+    grep
+    echo
+    bash
+commands =
+    bash -c 'grep -EIr -e "\{\{ ansible_managed \}\}" -e "# \{\{ ansible_managed" . &&
```

Comment:
each line must end with a `\` character or you will get this:
```
...
    raise tox.exception.ConfigError(
tox.exception.ConfigError: ConfigError: ansible-managed-var-comment failed with No closing quotation at Traceback (most recent call last):
...
  File "/usr/lib64/python3.9/shlex.py", line 191, in read_token
    raise ValueError("No closing quotation")
ValueError: No closing quotation
```
You also should exclude going into the `.tox` directory because otherwise you will get a lot of false positives:
```
./.tox/ansible-plugin-scan/lib/python3.9/site-packages/ansible_collections/community/proxysql/roles/proxysql/templates/client.my.cnf.j2:1:### {{ ansible_managed }}
...
./.tox/ansible-plugin-scan/lib/python3.9/site-packages/ansible_collections/google/cloud/roles/stackdriver_logging/templates/default/google-fluentd:1:# {{ ansible_managed }}
The ansible_managed variable must be commented with {{ ansible_managed | comment }} as described in https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#manipulating-text
ERROR: InvocationError for command /usr/bin/bash -c 'grep -EIrn -e '"'"'{{ ansible_managed }}'"'"' -e '"'"'# {{ ansible_managed'"'"' . &&  echo '"'"'The ansible_managed variable must be commented with {{ ansible_managed | comment }} as described in https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#manipulating-text'"'"' && exit 1 || exit 0' (exited with code 1)
```

Reply:
I've changed the grepped directory from `.` to `./*` so that grep searches non-hidden directories only.